en/admin/backup.inc

```diff
@@ -89,11 +89,11 @@ The following shows options available with the **cubrid backupdb** utility (opti
 
         cubrid backupdb -t 4 demodb
 
-.. option:: -z, --compress
+.. option:: --no-compress
 
-    The following shows how to compress the database and stores it in the backup file by using the **-z** option. The size of the backup file and the time required for backup can be reduced by using the **-z** option. ::
+    Basically, in order to reduce the size of the backup volume, CUBRID compresses the target database and then saves it. If the user wants to eliminate the CPU usage that occurs when the backup file is compressed, the backup volume compression can be stopped by setting the --no-compress option. ::
```

Comment:
```suggestion
    Basically, in order to reduce the size of the backup volume, CUBRID compresses the target database and then saves it. If the user wants to eliminate the CPU usage that occurs when the backup file is compressed, the backup volume won't compress by setting the --no-compress option. ::
```

Reply:
Opinion
```suggestion
    By default, CUBRID compresses the target database in order to reduce the size of the backup volume. If the user wants to eliminate the CPU usage that occurred by the compression, they can set the --no-compress option. ::
```

Reply:
I agree to avoid passive syntax and replace `By default` instead of `Basically`.
So, I can summarize your opinions.
```suggestion
    By default, CUBRID compresses the target database in order to reduce the size of the backup volume and then saves it. If the user wants to eliminate the CPU usage by the compression, they can set the --no-compress option not to compress the backup volume. ::
```

Reply:
LGTM. 
one thing: the user wants -> users want (or a user wants)